examples/step-47/doc/results.dox

```diff
@@ -211,3 +211,81 @@ make sense:
   addition should not be overly difficult using, for example, the
   FEInterfaceValues class combined with MeshWorker::mesh_loop() in the
   same spirit as we used for the assembly of the linear system.
+
+
+  <h3> Derivation for the simply supported plates </h3>
```

Comment:
Make that into an `h4` so that it shows up as a sub-section under "Possibilities for extensions".